tsconfig.json

```diff
@@ -66,7 +66,6 @@
       "tinymce/plugins/table/*": ["modules/tinymce/src/plugins/table/main/ts/*"],
       "tinymce/plugins/template/*": ["modules/tinymce/src/plugins/template/main/ts/*"],
       "tinymce/plugins/textcolor/*": ["modules/tinymce/src/plugins/textcolor/main/ts/*"],
-      "tinymce/plugins/textpattern/*": ["modules/tinymce/src/plugins/textpattern/main/ts/*"],
       "tinymce/plugins/toc/*": ["modules/tinymce/src/plugins/toc/main/ts/*"],
```

Comment:
Oops I must have missed this in a previous PR :cry: 
```suggestion
```
Just adding a suggestion in case there are other changes.

Reply:
I wonder how I did this... Bad merge I guess. Thanks!

Reply:
To clarify, you didn't do anything here and instead I just plain missed it in the previous PR :disappointed: